scripts/reset.js

```diff
@@ -0,0 +1,26 @@
+const ADB = require('appium-adb');
+const B = require('bluebird');
+const { flatMap } = require('lodash');
+
+const SERVER_PKGS = [
+  'io.appium.uiautomator2.server',
+  'io.appium.uiautomator2.server.test',
+  'io.appium.settings',
+];
+
+async function runReset () {
+  const adb = new ADB();
+  const udids = (await adb.getConnectedDevices())
+    .filter(({state}) => state === 'device')
+    .map(({udid}) => udid);
+  if (0 === udids.length) {
+    return;
```

Comment:
maybe we should log a warning like `Warning: no connected devices found. Reset not run`?

Reply:
We have adb logging the stuff for us:

```
% npm run reset

> appium-uiautomator2-driver@2.5.0 reset
> node ./scripts/reset.js

dbug ADB Getting connected devices
dbug ADB No connected devices have been detected
```